kibbeh/src/modules/admin/AdminPageForm.tsx

```diff
@@ -0,0 +1,122 @@
+import { UserWithFollowInfo, wrap } from "@dogehouse/kebab";
+import { useRouter } from "next/router";
+import React, { useEffect, useState } from "react";
+import { showErrorToast } from "../../lib/showErrorToast";
+import { useConn } from "../../shared-hooks/useConn";
+import { useTypeSafeTranslation } from "../../shared-hooks/useTypeSafeTranslation";
+import { Button } from "../../ui/Button";
+import { Input } from "../../ui/Input";
+import { MiddlePanel } from "../layouts/GridPanels";
+
+interface SearchUsersProps {}
+
+export const AdminPageForm: React.FC<SearchUsersProps> = ({}) => {
+  const conn = useConn();
+  const [username, setUsername] = useState("");
+  const [reason, setReason] = useState("");
+  const [contributions, setContributions] = useState(0);
+  const [isStaff, setIsStaff] = useState(false);
+  const [user, setUser] = useState<UserWithFollowInfo>();
+  const { t } = useTypeSafeTranslation();
+  const { replace } = useRouter();
+  const wrapper = wrap(conn);
+
+  useEffect(() => {
+    if (conn.user.username !== "benawad") {
+      showErrorToast("nice try");
+      replace("/dash");
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  useEffect(() => {
```

Comment:
don't worry about fetching the user

Reply:
i set that so you can see if the user is currently staff/contributor or not, otherwise it's hard to know

Reply:
I can just go to their profile if I need to know beforehand. I don't want to fetch on each keystroke